query-string/query-string.d.ts

```diff
@@ -9,14 +9,14 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): any;
+    export function parse(str: string): {[key: string]: string};
 
     /**
      * Stringify an object into a query string, sorting the keys.
      *
      * @param obj
      */
-    export function stringify(obj: any, options?: {strict: boolean}): string;
+    export function stringify(obj: {[key: string]: string|number|boolean}, options?: { strict: boolean }): string;
```

Comment:
It won't pass this example of `queryString.stringify({color: ['taupe', 'chartreuse'], id: '515'});`
I suggeust `function stringify(obj: {[key: string]: any| any[]}, options?: { strict: boolean }): string;`